src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/ManualInstructionModel.cs

```diff
@@ -11,17 +11,24 @@ namespace Microsoft.TemplateEngine.Orchestrator.RunnableProjects
     /// </summary>
     internal sealed class ManualInstructionModel : ConditionedConfigurationElementBase
     {
-        public ManualInstructionModel(string text)
+        public ManualInstructionModel(string? id, string text)
```

Comment:
It is nullable, because user doesn't have to specify an id when there is only one manual instruction.
Technically, whitespace or empty string is also fine for us. Basically, as long as they are unique we don't care much about the content of the string.

Reply:
okay, it's reasonable. The comment on the property will be helpful, to state the case when it's null.